### DISCUSSION TITLE
Binding of a small molecule to albumin AND alpha1-acid glycoprotein simultaneously

### DISCUSSION THREAD

**Discussion by nataliariva** [Open-Systems-Pharmacology/Forum#1706] [How-Tos]
Dear all, 
How can I add a protein binding of a compound to albumin **and** alpha1-acid glycoprotein **simultaneously** in PK-sim?
Is it better to choose "unknown"?
I have KD values for both bindings:
to Albumin: Kd=45650nM
to AAG: Kd= 83nM
![Captura de pantalla 2024-02-13 101606](https://github.com/Open-Systems-Pharmacology/Forum/assets/159889433/c303c0c6-ec96-47a4-963d-fe597342a291)
Thank you in advance for your help

**Comment by Yuri05** [comment]
You can add both proteins explicitly and define a binding reaction for each:
1. Add expression profile for Albumin
  i. Set the reference concentration of albumin (in plasma) (if known)
  ii. Set the relative expression in plasma = 1
  iii. Optionally import albumin ontogeny (if known)
<img width="450" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/d299a87c-a7c7-4e8a-a015-04ad6c390af9">

2. Repeat for AGP
3. Add both proteins to your individual or simulation
<img width="400" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/2b5c12d3-18eb-4914-a2e9-a257096c2eaa">

4. In your compound:
  i. Set fraction unbound = 1 and binding partner = "Unknown"
  ii. Add both proteins as binding partners and enter your Kd values
<img width="500" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/d447ba58-36d2-4c7a-aa25-b58fa0e4830b">

<img width="500" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/b397ec18-0273-4e44-b539-f0da803e0810">

5. Make sure to map both proteins when creating a simulation
<img width="350" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/7c6ab74c-80be-4100-ab5f-b444dca00050">

**Reply by nataliariva** [reply]
Thank you for your response Juri ! I have added the expression profile of both proteins as you suggested.
In my compound I set fraction unbound = 1 and binding partner = "Unknown"
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/159889433/9af0a4c8-f758-40a0-afcf-f71731c7d1bd)

In the individual I added both proteins
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/159889433/6b5eff57-d265-4b45-9d0c-0dd9c471631e)

and included the binding partners in the compound
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/159889433/349f4422-7e04-494d-8c94-9e707c707a1f)

I performed the simulation mapping both proteins
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/159889433/d7ddc0eb-ff1b-4562-930a-f6817a2cfa82)

I obtain the following profile
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/159889433/11434566-f369-44d6-ad05-8b4c91a21d4b)

I interpret this as the system is not considering the included binding to both proteins and only considering the Fu=1 (unknown), given the different profile that I obtain if I change that parameter in the compound and repeat the simulation
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/159889433/54b67b5e-033c-4957-b121-ddecbffb170c)
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/159889433/bfc88724-c807-447f-8bca-617c42d46a10)

Since the compound is all unbound (Fu=1), for this drug disposition parameters (V, Cl) are higher and simulated concentrations do not reach the levels of expected observations. However, after changing that value, predictions increased, Is it possible? How can I effectively correct this and add the binding?
Thank you for your help again!
Best regards

**Reply by Yuri05** [reply]
Is it possible to upload your project?

**Reply by nataliariva** [reply]
Dear Yuri, thank you and sorry for my late response. I have been discussing practical issues, and finally I am able to upload a file with the dummy example.

**Reply by nataliariva** [reply]
[TEST_OSP.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/15198421/TEST_OSP.zip)

**Reply by nataliariva** [reply]
Dear OSP community, I have not received an answer, can someone help me?
